models/BaseConfig.cfc

```diff
@@ -1103,6 +1107,49 @@ component accessors="true" {
 		return this;
 	}
 
+
+	/**
+	* Add a single component Path to the config
+	*
+	* @physical The physical path that the engine should search
+	* @archive Path to the Lucee/Railo archive
+	* @name Name of the Component Path
+	* @inspectTemplate String containing one of "never", "once", "always", "" (inherit)
+	* @primary Strings containing one of "physical", "archive"
+
+
+		inspect-template	
+		physical	
+		primary	
+		readonly	
+		virtual	
+
+
+	*/
+	function addComponentPath(
+			required string name,
+			required string physical,
+			string archive="",
+			string primary="physical",
+			string inspectTemplate="never",
+			boolean readOnly=false,
```

Comment:
User added component mappings don't appear to ever use the readonly attribute, so I think we can remove this.